Sources/Orbit/Support/Text/TextBuildable.swift

```diff
@@ -1,41 +1,331 @@
 import SwiftUI
 
+// Modifiers that shadow the SwiftUI modifiers need to have the exact signature
+// in order to prefer the method of the Text over the generic View version
```

Comment:
So this exists in order for these modifierrs to return the same type (`Self`)?

Reply:
Yes exactly - just as the native `Text` does. Otherwise, it would be `just` a View. This makes it possible to build the `Text` properties and later concatenate them using the same syntax as the native one.